waku/v2/node/storage/message/waku_message_store.nim

```diff
@@ -61,7 +61,7 @@ method put*(db: WakuMessageStore, cursor: Index, message: WakuMessage, pubsubTop
   ## 
   let prepare = db.database.prepareStmt(
     "INSERT INTO " & TABLE_TITLE & " (id, receiverTimestamp, contentTopic, payload, pubsubTopic, version, senderTimestamp) VALUES (?, ?, ?, ?, ?, ?, ?);",
-    (seq[byte], float64, seq[byte], seq[byte], seq[byte], int64, float64),
+    (seq[byte], int64, seq[byte], seq[byte], seq[byte], int64, int64),
     void
   )
 
```

Comment:
Lines 94 and 110 of this file also need change (The receiver timestamp and sender timestamp should be treated as int not double)

```
receiverTimestamp = sqlite3_column_double(s, 0)
```

```
senderTimestamp = sqlite3_column_double(s, 5)
```

Reply:
In light of the new type Timestamp definition, is it worth to define a sqlite3_column_timestamp?

Reply:
No need for that, the underlying `sqlite3_gen` already has support for `sqlite3_column_int64`

Reply:
Using `sqlite3_column_int64` might partially defeat the changes proposed in [this](https://github.com/status-im/nim-waku/pull/842#discussion_r801149390) comment. I defined a `column_timestamp` which calls `sqlite3_column_int64` in utils/time.